### ISSUE TITLE
CAPS LOCK Broken in 0.90.68

### ISSUE THREAD

**Issue by rnbarg** [tekezo/Karabiner-Elements#505]
The CAPS LOCK button no longer works with 0.90.68.  The light on the key doesn't work and the key itself does not work.

Can I return to an earlier version in the interim.

**Comment by 0x3333** [comment]
Same issue here.
Here is my log:

```
[2016-12-15 22:37:37.288] [info] [console_user_server] grabber_client_ is connected
[2016-12-15 22:37:37.342] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-15 22:37:37.344] [warning] [grabber] event_manipulator_ is not ready. (event_dispatcher_manager_is_not_ready is not ready) Please wait for a while.
[2016-12-15 22:37:38.019] [warning] [grabber] event_manipulator_ is not ready. (event_dispatcher_manager_is_not_ready is not ready) Please wait for a while.
[2016-12-15 22:37:38.291] [info] [grabber] system_preferences_values_updated
[2016-12-15 22:37:38.291] [info] [console_user_server] system_preferences::values is updated.
[2016-12-15 22:37:38.313] [info] [grabber] karabiner_event_dispatcher is launched. (pid:719)
[2016-12-15 22:37:38.343] [info] [grabber] karabiner_event_dispatcher is connected (pid:719)
[2016-12-15 22:37:38.344] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-15 22:37:38.434] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-16 09:05:22.317] [info] [grabber] removal device:
[2016-12-16 09:05:22.319] [info] [grabber]   vendor_id: 0x5ac
[2016-12-16 09:05:22.319] [info] [grabber]   product_id: 0x290
[2016-12-16 09:05:22.320] [info] [grabber]   location_id: 0x1000000
[2016-12-16 09:05:22.326] [info] [grabber] matching device:
[2016-12-16 09:05:22.326] [info] [grabber]   product: Apple Internal Keyboard / Trackpad
[2016-12-16 09:05:22.326] [info] [grabber]   vendor_id: 0x5ac
[2016-12-16 09:05:22.326] [info] [grabber]   product_id: 0x290
[2016-12-16 09:05:22.326] [info] [grabber]   location_id: 0x1000000
[2016-12-16 09:05:22.326] [info] [grabber]   registry_entry_id: 4294968733
[2016-12-16 09:05:22.525] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-16 10:49:24.147] [info] [grabber] current_console_user_id: 0
[2016-12-16 10:49:24.321] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:719)
[2016-12-16 10:49:24.517] [info] [grabber] Connected devices are ungrabbed
[2016-12-16 10:49:24.517] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-16 10:49:25.518] [info] [grabber] karabiner_event_dispatcher is launched. (pid:3459)
[2016-12-16 10:49:25.543] [info] [grabber] karabiner_event_dispatcher is connected (pid:3459)
[2016-12-16 10:49:25.543] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-16 11:04:41.162] [info] [grabber] current_console_user_id: 502
[2016-12-16 11:04:41.196] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:3459)
[2016-12-16 11:04:41.403] [warning] [console_user_server] grabber socket is not writable
[2016-12-16 11:04:41.949] [warning] [console_user_server] grabber socket is not found
[2016-12-16 11:04:42.162] [info] [grabber] Connected devices are ungrabbed
[2016-12-16 11:04:42.162] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-16 11:04:42.445] [info] [grabber] karabiner_event_dispatcher is launched. (pid:3693)
[2016-12-16 11:04:42.481] [info] [grabber] karabiner_event_dispatcher is connected (pid:3693)
[2016-12-16 11:04:42.481] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-16 11:04:42.874] [info] [console_user_server] grabber_client_ is connected
[2016-12-16 11:04:42.880] [info] [grabber] karabiner_console_user_server is connected (pid:692)
[2016-12-16 11:04:42.890] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-16 11:04:43.085] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-16 11:04:43.932] [info] [grabber] system_preferences_values_updated
[2016-12-16 11:04:43.932] [info] [console_user_server] system_preferences::values is updated.
[2016-12-16 13:55:06.006] [info] [grabber] current_console_user_id: 0
[2016-12-16 13:55:06.233] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:3693)
[2016-12-16 13:55:06.393] [info] [grabber] Connected devices are ungrabbed
[2016-12-16 13:55:06.393] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-16 13:55:07.442] [info] [grabber] karabiner_event_dispatcher is launched. (pid:5445)
[2016-12-16 13:55:07.542] [info] [grabber] karabiner_event_dispatcher is connected (pid:5445)
[2016-12-16 13:55:07.542] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-16 13:55:10.948] [info] [grabber] current_console_user_id: 501
[2016-12-16 13:55:11.001] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:5445)
[2016-12-16 13:55:11.736] [info] [grabber] Connected devices are ungrabbed
[2016-12-16 13:55:11.742] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-16 13:55:12.106] [info] [grabber] karabiner_event_dispatcher is launched. (pid:5499)
[2016-12-16 13:55:13.094] [info] [grabber] karabiner_event_dispatcher is connected (pid:5499)
[2016-12-16 13:55:13.094] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-16 14:34:53.128] [info] [grabber] current_console_user_id: 0
[2016-12-16 14:34:53.218] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:5499)
[2016-12-16 14:34:53.441] [info] [grabber] Connected devices are ungrabbed
[2016-12-16 14:34:53.441] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-16 14:34:54.359] [info] [grabber] karabiner_event_dispatcher is launched. (pid:7224)
[2016-12-16 14:34:54.510] [info] [grabber] karabiner_event_dispatcher is connected (pid:7224)
[2016-12-16 14:34:54.511] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-16 14:47:33.793] [info] [grabber] current_console_user_id: 502
[2016-12-16 14:47:33.820] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:7224)
[2016-12-16 14:47:33.843] [warning] [console_user_server] grabber socket is not found
[2016-12-16 14:47:34.273] [warning] [console_user_server] grabber socket is not found
[2016-12-16 14:47:34.787] [info] [grabber] Connected devices are ungrabbed
[2016-12-16 14:47:34.787] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-16 14:47:34.854] [info] [grabber] karabiner_event_dispatcher is launched. (pid:7298)
[2016-12-16 14:47:34.876] [info] [grabber] karabiner_event_dispatcher is connected (pid:7298)
[2016-12-16 14:47:34.876] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-16 14:47:35.257] [info] [grabber] karabiner_console_user_server is connected (pid:692)
[2016-12-16 14:47:35.257] [info] [console_user_server] grabber_client_ is connected
[2016-12-16 14:47:35.276] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-16 14:47:35.408] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-16 14:47:36.258] [info] [console_user_server] system_preferences::values is updated.
[2016-12-16 14:47:36.259] [info] [grabber] system_preferences_values_updated
[2016-12-18 11:29:39.660] [info] [grabber] removal device:
[2016-12-18 11:29:41.277] [info] [grabber]   vendor_id: 0x5ac
[2016-12-18 11:29:41.277] [info] [grabber]   product_id: 0x290
[2016-12-18 11:29:41.277] [info] [grabber]   location_id: 0x1000000
[2016-12-18 11:29:42.436] [info] [grabber] matching device:
[2016-12-18 11:29:42.436] [info] [grabber]   product: Apple Internal Keyboard / Trackpad
[2016-12-18 11:29:42.437] [info] [grabber]   vendor_id: 0x5ac
[2016-12-18 11:29:42.437] [info] [grabber]   product_id: 0x290
[2016-12-18 11:29:42.437] [info] [grabber]   location_id: 0x1000000
[2016-12-18 11:29:42.437] [info] [grabber]   registry_entry_id: 4294971731
[2016-12-18 11:29:42.914] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-19 09:07:26.807] [info] [grabber] removal device:
[2016-12-19 09:07:27.051] [info] [grabber]   vendor_id: 0x5ac
[2016-12-19 09:07:27.058] [info] [grabber]   product_id: 0x290
[2016-12-19 09:07:27.058] [info] [grabber]   location_id: 0x1000000
[2016-12-19 09:07:27.146] [info] [grabber] matching device:
[2016-12-19 09:07:27.146] [info] [grabber]   product: Apple Internal Keyboard / Trackpad
[2016-12-19 09:07:27.148] [info] [grabber]   vendor_id: 0x5ac
[2016-12-19 09:07:27.148] [info] [grabber]   product_id: 0x290
[2016-12-19 09:07:27.148] [info] [grabber]   location_id: 0x1000000
[2016-12-19 09:07:27.148] [info] [grabber]   registry_entry_id: 4294971969
[2016-12-19 09:07:27.359] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-19 09:07:29.093] [info] [grabber] matching device:
[2016-12-19 09:07:29.093] [info] [grabber]   manufacturer: 
[2016-12-19 09:07:29.093] [info] [grabber]   product: USB Receiver
[2016-12-19 09:07:29.093] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:07:29.093] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:07:29.093] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:07:29.093] [info] [grabber]   registry_entry_id: 4294972137
[2016-12-19 09:07:29.228] [warning] [grabber] We cannot grab Apple Internal Keyboard / Trackpad while a key is repeating.
[2016-12-19 09:07:29.296] [info] [grabber] USB Receiver is grabbed
[2016-12-19 09:08:02.777] [info] [grabber] removal device:
[2016-12-19 09:08:02.777] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:08:02.777] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:08:02.777] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:08:05.835] [info] [grabber] matching device:
[2016-12-19 09:08:05.835] [info] [grabber]   manufacturer: 
[2016-12-19 09:08:05.835] [info] [grabber]   product: USB Receiver
[2016-12-19 09:08:05.835] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:08:05.835] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:08:05.836] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:08:05.836] [info] [grabber]   registry_entry_id: 4294972364
[2016-12-19 09:08:06.112] [info] [grabber] USB Receiver is grabbed
[2016-12-19 09:10:46.984] [info] [grabber] removal device:
[2016-12-19 09:10:46.992] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:10:46.992] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:10:46.992] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:14:03.944] [info] [grabber] matching device:
[2016-12-19 09:14:03.944] [info] [grabber]   manufacturer: 
[2016-12-19 09:14:03.944] [info] [grabber]   product: USB Receiver
[2016-12-19 09:14:03.944] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:14:03.944] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:14:03.944] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:14:03.944] [info] [grabber]   registry_entry_id: 4294972753
[2016-12-19 09:14:04.179] [info] [grabber] USB Receiver is grabbed
[2016-12-19 09:16:05.617] [info] [grabber] removal device:
[2016-12-19 09:16:05.618] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:16:05.618] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:16:05.618] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:16:06.547] [info] [grabber] matching device:
[2016-12-19 09:16:06.547] [info] [grabber]   manufacturer: 
[2016-12-19 09:16:06.547] [info] [grabber]   product: USB Receiver
[2016-12-19 09:16:06.547] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:16:06.547] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:16:06.547] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:16:06.548] [info] [grabber]   registry_entry_id: 4294972986
[2016-12-19 09:16:06.753] [info] [grabber] USB Receiver is grabbed
[2016-12-19 09:26:47.093] [info] [grabber] removal device:
[2016-12-19 09:26:47.099] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:26:47.099] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:26:47.099] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:26:49.366] [info] [grabber] matching device:
[2016-12-19 09:26:49.366] [info] [grabber]   manufacturer: 
[2016-12-19 09:26:49.366] [info] [grabber]   product: USB Receiver
[2016-12-19 09:26:49.366] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 09:26:49.366] [info] [grabber]   product_id: 0xc52b
[2016-12-19 09:26:49.366] [info] [grabber]   location_id: 0x14130000
[2016-12-19 09:26:49.366] [info] [grabber]   registry_entry_id: 4294973304
[2016-12-19 09:26:49.549] [warning] [grabber] We cannot grab USB Receiver while a key is repeating.
[2016-12-19 09:26:49.672] [info] [grabber] USB Receiver is grabbed
[2016-12-19 10:28:59.035] [info] [grabber] current_console_user_id: 0
[2016-12-19 10:28:59.244] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:7298)
[2016-12-19 10:29:00.053] [info] [grabber] Connected devices are ungrabbed
[2016-12-19 10:29:00.053] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-19 10:29:00.570] [info] [grabber] karabiner_event_dispatcher is launched. (pid:17230)
[2016-12-19 10:29:00.749] [info] [grabber] karabiner_event_dispatcher is connected (pid:17230)
[2016-12-19 10:29:00.749] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-19 10:33:54.175] [info] [grabber] removal device:
[2016-12-19 10:33:54.189] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 10:33:54.189] [info] [grabber]   product_id: 0xc52b
[2016-12-19 10:33:54.189] [info] [grabber]   location_id: 0x14130000
[2016-12-19 10:34:51.845] [info] [grabber] current_console_user_id: 502
[2016-12-19 10:34:51.858] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:17230)
[2016-12-19 10:34:52.026] [warning] [console_user_server] grabber socket is not found
[2016-12-19 10:34:53.397] [info] [grabber] karabiner_event_dispatcher is launched. (pid:17380)
[2016-12-19 10:34:53.398] [warning] [console_user_server] grabber socket is not found
[2016-12-19 10:34:53.436] [info] [grabber] Connected devices are ungrabbed
[2016-12-19 10:34:53.436] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-19 10:34:53.596] [info] [grabber] karabiner_event_dispatcher is connected (pid:17380)
[2016-12-19 10:34:53.596] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-19 10:34:53.903] [info] [grabber] karabiner_console_user_server is connected (pid:692)
[2016-12-19 10:34:53.903] [info] [console_user_server] grabber_client_ is connected
[2016-12-19 10:34:53.911] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-19 10:34:54.103] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-19 10:34:54.980] [info] [grabber] system_preferences_values_updated
[2016-12-19 10:34:54.980] [info] [console_user_server] system_preferences::values is updated.
[2016-12-19 11:08:07.585] [info] [grabber] pid:692 is exited
[2016-12-19 11:08:07.752] [info] [grabber] Connected devices are ungrabbed
[2016-12-19 11:09:34.782] [info] [console_user_server] version 0.90.68
[2016-12-19 11:09:39.591] [info] [console_user_server] grabber_client_ is connected
[2016-12-19 11:09:39.616] [info] [grabber] karabiner_console_user_server is connected (pid:18617)
[2016-12-19 11:09:39.671] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-19 11:09:39.905] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-19 11:09:40.724] [info] [console_user_server] system_preferences::values is updated.
[2016-12-19 11:09:40.779] [info] [grabber] system_preferences_values_updated
[2016-12-19 11:27:42.146] [info] [grabber] matching device:
[2016-12-19 11:27:42.146] [info] [grabber]   manufacturer: 
[2016-12-19 11:27:42.146] [info] [grabber]   product: USB Receiver
[2016-12-19 11:27:42.146] [info] [grabber]   vendor_id: 0x46d
[2016-12-19 11:27:42.148] [info] [grabber]   product_id: 0xc52b
[2016-12-19 11:27:42.148] [info] [grabber]   location_id: 0x14130000
[2016-12-19 11:27:42.148] [info] [grabber]   registry_entry_id: 4294974681
[2016-12-19 11:27:42.539] [info] [grabber] USB Receiver is grabbed
[2016-12-19 12:44:05.307] [info] [grabber] current_console_user_id: 0
[2016-12-19 12:44:05.513] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:17380)
[2016-12-19 12:44:06.471] [info] [grabber] Connected devices are ungrabbed
[2016-12-19 12:44:06.471] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-19 12:44:06.686] [info] [grabber] karabiner_event_dispatcher is launched. (pid:21303)
[2016-12-19 12:44:07.105] [info] [grabber] karabiner_event_dispatcher is connected (pid:21303)
[2016-12-19 12:44:07.105] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-19 13:43:41.226] [warning] [console_user_server] grabber socket is not writable
[2016-12-19 13:43:42.255] [info] [grabber] current_console_user_id: 502
[2016-12-19 13:43:42.266] [warning] [console_user_server] grabber socket is not writable
[2016-12-19 13:43:42.270] [info] [grabber] karabiner_event_dispatcher is terminated. (pid:21303)
[2016-12-19 13:43:43.192] [info] [grabber] Connected devices are ungrabbed
[2016-12-19 13:43:43.192] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-19 13:43:43.234] [info] [console_user_server] grabber_client_ is connected
[2016-12-19 13:43:43.253] [info] [grabber] karabiner_console_user_server is connected (pid:18617)
[2016-12-19 13:43:43.253] [error] [grabber] client_ is not ready @ void manipulator::event_dispatcher_manager::set_caps_lock_state(bool)
[2016-12-19 13:43:43.277] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-19 13:43:43.301] [warning] [grabber] event_manipulator_ is not ready. (event_dispatcher_manager_is_not_ready is not ready) Please wait for a while.
[2016-12-19 13:43:43.404] [info] [grabber] karabiner_event_dispatcher is launched. (pid:22738)
[2016-12-19 13:43:43.685] [info] [event_dispatcher] version 0.90.68
[2016-12-19 13:43:43.798] [info] [event_dispatcher] IOServiceOpen is succeeded @ void hid_system_client::matched_callback(io_iterator_t)
[2016-12-19 13:43:43.807] [info] [grabber] karabiner_event_dispatcher is connected (pid:22738)
[2016-12-19 13:43:43.807] [info] [grabber] karabiner_event_dispatcher client is created.
[2016-12-19 13:43:43.820] [info] [event_dispatcher] receiver is started
[2016-12-19 13:43:43.842] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-19 13:43:43.851] [info] [grabber] USB Receiver is grabbed
[2016-12-19 13:43:44.237] [info] [grabber] system_preferences_values_updated
[2016-12-19 13:43:44.237] [info] [console_user_server] system_preferences::values is updated.
[2016-12-19 14:52:26.031] [info] [grabber] pid:18617 is exited
[2016-12-19 14:52:26.087] [info] [grabber] Connected devices are ungrabbed
[2016-12-19 14:53:07.868] [info] [console_user_server] version 0.90.68
[2016-12-19 14:53:09.276] [info] [console_user_server] grabber_client_ is connected
[2016-12-19 14:53:09.277] [info] [grabber] karabiner_console_user_server is connected (pid:24981)
[2016-12-19 14:53:09.277] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-19 14:53:09.442] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-19 14:53:09.454] [info] [grabber] USB Receiver is grabbed
[2016-12-19 14:53:10.295] [info] [console_user_server] system_preferences::values is updated.
[2016-12-19 14:53:10.296] [info] [grabber] system_preferences_values_updated
[2016-12-19 14:54:27.730] [info] [grabber] pid:24981 is exited
[2016-12-19 14:54:27.732] [info] [grabber] Connected devices are ungrabbed
[2016-12-19 14:54:32.994] [info] [console_user_server] version 0.90.68
[2016-12-19 14:54:34.286] [info] [grabber] karabiner_console_user_server is connected (pid:25025)
[2016-12-19 14:54:34.286] [info] [console_user_server] grabber_client_ is connected
[2016-12-19 14:54:34.287] [info] [console_user_server] core_configuration_ was loaded.
[2016-12-19 14:54:34.475] [info] [grabber] Apple Internal Keyboard / Trackpad is grabbed
[2016-12-19 14:54:34.478] [info] [grabber] USB Receiver is grabbed
[2016-12-19 14:54:35.309] [info] [grabber] system_preferences_values_updated
[2016-12-19 14:54:35.309] [info] [console_user_server] system_preferences::values is updated.
```

**Comment by 0x3333** [comment]
A work around is to quit karabiner and open it again.

**Comment by cmbrandenburg** [comment]
I too began having this problem immediately after installing the MacOS update 10.12.2. Before installing the update, my caps lock key worked fine.

The physical key works okay, as I can use Karabiner-Elements to remap `caps_lock` to some other key, e.g., `return_or_enter`, and pressing the caps lock key has the effect of pressing the target key. But without a remapping, the caps lock key has no effect.

While Karabiner-Elements is open, the green caps lock light never turns on.

Also, I can remap other keys to `caps_lock`, and those keys have no effect while the remapping is in effect. It's as if the *logical* functionality of caps lock is gone.

Quitting Karabiner-Elements causes the caps lock key to work correctly again. Restarting Karabiner-Elements causes the caps lock key to stop working. I.e., quitting Karabiner-Elements and restarting it does **not** work around the issue.

**Comment by cmbrandenburg** [comment]
I downgraded Karabiner-Elements to version 0.90.55—the only other version for which I have an installer—and my caps locks key now works correctly.

**Comment by 0x3333** [comment]
I'm on version 0.90.68. I'll try to downgrade to see what happens.

**Comment by nickcurran** [comment]
Same issue with num_lock, which you wouldn't normally see unless you were trying it in Windows via VirtualBox or the like.

**Comment by pilot34** [comment]
Caps lock didn't work after update for me too. It was constant error in the Karabiner log for me:
`grabber socket is not writable`

I went to the sockets folder
`cd /Library/Application\ Support/org.pqrs/tmp`
And all files were not writable for my current user, only were writable for root.

I executed (replace **your_macos_username** with your user name):
`sudo chown your_macos_username karabiner_*`
Restarted Karabiner and everything begins to work.
